docs/make.jl

```diff
@@ -73,6 +73,7 @@ makedocs(modules = [PositiveIntegrators],
          # Explicitly specify documentation structure
          pages = [
              "Home" => "index.md",
+             "Tutorials" => "linear_advection.md",
```

Comment:
```suggestion
             "Tutorials" => [
                 "Linear Advection" => "linear_advection.md",
             ],
```

This makes a list of tutorials that we can extend step by step.